### PR TITLE
v1.32.0 Add checkbox icons which are needed for Favourites

### PR DESCRIPTION
**Pull request by tommcclean**
### Added
- Added checkbox icons  (checked and unchecked)

### CODE REVIEW

File: src/img/icons/forms/checkbox--checked.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M11 17l-5-5.299 1.399-1.43 3.574 3.736L17.545 7 19 8.403 11 17zM22 2v20H2V2h20zm2-2H0v24h24V0z"/></svg>
```
Comment (DamianMullins) [inline src/img/icons/forms/checkbox--checked.svg]: These icons look they're missing a border for the right-hand side and bottom, is that right?

Comment (tommcclean) [reply src/img/icons/forms/checkbox--checked.svg]: Good catch, that svomg tool removed them

Comment (tommcclean) [reply src/img/icons/forms/checkbox--checked.svg]: Do we tend to get these icons from a standard place, or do we make them?

Comment (DamianMullins) [reply src/img/icons/forms/checkbox--checked.svg]: We've handled checkboxes etc with CSS in the past - https://fozzie.just-eat.com/styleguide/ui-components/forms/checkboxes

Comment (tommcclean) [reply src/img/icons/forms/checkbox--checked.svg]: Right, thanks for that, ill do that

**Pull request closed without merging**